dev/visualization/visualize_semantic_labels.py

```diff
@@ -26,6 +26,8 @@ def parse_args():
         default=0.5,
         help="Contribution of first image to blended one",
     )
+    parser.add_argument("--ignore-substr-images-for-matching", default="", type=str)
```

Comment:
Nice job adding the appropriate default here so it would not break the function if the argument wasn't provided. Just so you know, if you don't specify a default value, the default is `None`. Also, the default type is `str` so you could remove this argument if you wanted.